api/footnotes.asciidoc

```diff
@@ -53,7 +53,7 @@ Note that reading and writing 2D image arrays from a kernel with `image_array_si
 ]
 
 :fn-image-from-buffer: pass:n[ \
-To create a 2D image from a buffer object that share the data store between the image and buffer object. \
+To create an image from a buffer object that share the data store between the image and buffer object. \
```

Comment:
Optional / editorial: this wasn't changed in this PR, but I don't this sentence is grammatically correct.  How about:

```suggestion
To create an image from a buffer object that shares the data store between the image and buffer object. \
```